api_tests/registrations/views/test_registration_detail.py

```diff
@@ -655,23 +655,30 @@ def test_read_write_contributor_cannot_update_admin_writeable_fields(
             attributes={'description': 'Updated description'}
         )
         res = app.put_json_api(private_url, payload, auth=read_write_contributor.auth, expect_errors=True)
-        assert res.status_code == 403
+        assert res.status_code == 200
+
+        payload = make_payload(
+            id=private_registration._id,
+            attributes={'title': 'Updated title'}
+        )
+        res = app.put_json_api(private_url, payload, auth=read_write_contributor.auth, expect_errors=True)
+        assert res.status_code == 200
 
         #  test_read_write_contributor_cannot_update_category
         payload = make_payload(
             id=private_registration._id,
             attributes={'category': 'instrumentation'}
         )
         res = app.put_json_api(private_url, payload, auth=read_write_contributor.auth, expect_errors=True)
```

Comment:
Are you expecting errors after these changes?

Reply:
Nope I removed the expect_errors arguments.